### ISSUE TITLE
JSON Schema default value support

### ISSUE THREAD

**Issue by AChakraborty1982** [eclipsesource/jsonforms#570]
Does data Schema default value, is used during render?

Schema extract:

```Javascript
"isVisible": {
  "type": "boolean",
  "default":true
}
```
UI schema:

```Javascript
{
  "type": "Control",
  "label": "Is Visible",
  "scope": {
    "$ref": "#/properties/isVisible"
  }
}
```

**Comment by edgarmueller** [comment]
No, currently not, but supporting this would make sense. I suppose, that the default value would only be used in case the respective data property is unset, correct? E.g., in this example, when binding an empty object `{ }`, `default`'s value would be used, but when binding against `{ isVisible: false }` the default value would be ignored.

**Comment by AChakraborty1982** [comment]
Yes that would be useful.

**Comment by koegel** [comment]
This means opening a form on a JSON object will change the object without user interaction e.g. entering values into the form. I think this is not desirable. I would therefore propose a framework user should pre-initialize their JSON with default values before passing it into JSON Forms.

**Comment by eneufeld** [comment]
@edgarmueller isn't this fixed via #1193 ?

**Comment by edgarmueller** [comment]
It's not fixed, but we can close it as a duplicate of #1193
